src/main/java/step2/LadderGame.java

```diff
@@ -0,0 +1,23 @@
+package step2;
+
+import step2.model.Ladder;
+import step2.model.Person;
+import step2.view.InputView;
+import step2.view.OutputView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LadderGame {
+
+    public static void main(String[] args) {
+        List<Person> participants = new ArrayList<>();
+        String[] nameList = InputView.askNameForLadderGame();
+        for (String name : nameList) {
+            participants.add(new Person(name));
+        }
```

Comment:
Participants를 일급컬렉션으로 만들어보는건 어떨까요?
그리고 해당 코드는
```suggestion
        String[] nameList = InputView.askNameForLadderGame();
        List<Person> participants = Arrays.stream()
                                          .map(name -> Person::new);
```
이렇게 표현할 수 있습니다!